package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "storm-jsonrpc",
+  "version": "0.0.1",
+  "author": "Sreelal Nampoothiri <sreelalhn@gmail.com>",
+  "scripts": {
+    "start":"node -r esm src/storm_jsonrpc.js",	 
+    "lint": "eslint '**/*.js'",
+    "lint:fix": "eslint '**/*.js' --fix",
+    "postinstall": "(test -e testcases/Storm-Testcases || git clone https://github.com/WebPlatformForEmbedded/Storm-Testcases.git testcases/Storm-Testcases)"
+  },
+  "lint-staged": {
+    "*.js": [
+      "eslint --fix",
+      "git add"
+    ]
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged"
+    }
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/WebPlatformForEmbedded/Storm-JSONRPC.git"
+  },
+  "bugs": {
+    "url": "https://github.com/WebPlatformForEmbedded/Storm-JSONRPC/issues"
+  },
+  "homepage": "https://github.com/WebPlatformForEmbedded/Storm-JSONRPC#readme",
+  "dependencies": {
```

Comment:
I see some unused dependencies (chalk, contra, inquirer, moment)

Reply:
Thank you Michiel for the review.

We have removed the unused dependencies you have pointed out.